test/test_modules_before_internship_backend.py

```diff
@@ -17,26 +17,16 @@ class TestCode(object):
         query related functions.
     '''
     def __init__(self):
-        self.TEST_MOD_1 = 'CS6101'
-        self.TEST_MOD_2 = 'CP3200'
-        self.TEST_MOD_3 = 'CS3230'
-        self.TEST_MOD_4 = 'CS2105'
-        self.isDuringTesting = False # a boolean flag to allow functions to be called by
-                                     # other functions only, and not by nosetests
-
+        self.INTERN_MOD = 'CP3200'
+        self.INTERN_SEM = 'AY 17/18 Sem 1'
 
     def test_query_module_before_internship_empty(self):
         '''
             Tests querying the list of modules taken before internship,
             in AY/Sem which no one is doing internship in.
         '''
         list_of_modules_before_internship = model.get_mod_before_intern('AY 16/17 Sem 1')
```

Comment:
Can consider using your technique of adding a dummy entry, then testing if that entry should (or should not) be in a certain result set.